benchmark/python/benchmark_e2e.py

```diff
@@ -232,6 +240,9 @@ def run_benchmark(args, batch_size, prompt_length, generation_length, max_length
         # use random tokens instead of generating a prompt using the model and then tokenizing it
         tokens = np.random.randint(100, size=(batch_size, prompt_length))
         prompt = [tokenizer.decode(tokens[0])] * batch_size
+    elif args.use_prompt_set:
+        prompt = get_prompt_by_length(prompt_length)
+        tokens = tokenizer.encode_batch(prompt)
```

Comment:
Different tokenizers can encode prompts into different prompt lengths. Some additional work is needed to get the desired prompt length. You can see an example of how to do this [here](https://github.com/microsoft/onnxruntime/blob/00b262dbb4328173bf0e11cbcd1d62e18df3d686/onnxruntime/python/tools/transformers/models/llama/llama_inputs.py#L424-L441).

Reply:
so basically, we will check the `tokens` length against requested `prompt_length` and we will add/trim if it is needed. Is that correct?